spark_pipeline_framework/transformers/http_data_sender/v1/http_data_sender.py

```diff
@@ -188,10 +188,14 @@ def send_partition_to_server(
                 ]
                 # logger = get_logger(__name__)
                 if len(json_data_list) == 0:
+                    self.logger.info(f"Batch {partition_index} is empty")
```

Comment:
loggers don't work inside the map functions since that code is run on Spark worker nodes where self.logger is not available.

Reply:
We are currently using print statements inside the send_partition_to_server function for fhir sender transformer.
Would print statement work or is it there just for local testing?

Reply:
print statements do work since the logs are written on worker nodes and our log aggregator can pick them up.